common.h

```diff
@@ -88,8 +88,10 @@ constexpr auto SPRITEWIDTH = TILEWIDTH;
 constexpr auto SPRITEHEIGHT = (TILETOPHEIGHT + WALLHEIGHT);
 constexpr auto WALL_CUTOFF_HEIGHT = 15;
 
-constexpr auto DEFAULT_SIZE = 20;
-constexpr auto DEFAULT_SIZE_Z = 6;
+constexpr auto MIN_SEGSIZE = 1;
+constexpr auto MAX_SEGSIZE = 100;;
```

Comment:
why 100? that actually feels on the small side. Why have a maximum at all? We don't for SEGSIZE_Z, do we?